data/map planets.txt

```diff
@@ -205,21 +209,20 @@ planet Mikhailnov
 		fleet "Large Republic" 16
 
 planet Margad
-	attributes farming sanctuary frontier religious south
+	attributes farming sanctuary frontier religious south tourism
 	landscape land/nvalley1
-	description "The second planet inhabited by the Hindu people, Margad was settled as a spiritual home for Hinduism, as years passed the small settlement became a colony with millions of residents settling here. Recently it became a tourist spot and the many temples constructed receive thousands of visitors every year however there are still many more communities who are truly secluded from the rest of the world."
-	spaceport "The spaceport is on quite the distance from the capital, you can see a few buildings on the horizon but not much more. The port itself is a circular path with smaller circles acting as platforms to the starships, from the air it looks like a symbol but you cant recognize it."
-	spaceport "    At the center of the circle there's a terminal where people meet before its next destination. Many vehicles station outside come and go taking and droping people before going to the city. In terrace is a beautiful large garden where people can watch the ships take off.  "
-	government Republic
+	description "The second planet colonized by the Hindu people, Margad was settled as a new spiritual center for Hinduism. As years passed, the small settlement became a major colony with tens of millions of residents making their livelihoods here. Recently it became a tourist spot, and the many temples here constructed receive thousands of visitors every year. Despite this leap in connections with the rest of the galaxy, there are still many more communities who are truly secluded from the rest of the world."
+	spaceport "The spaceport is quite far from the capital; you can see a few buildings on the horizon, but not much more. The port itself is a circular building, with smaller circles acting as platforms to the starships. From the air it's obvious that the building was deliberately designed to resemble a symbol, but you can't recognize it."
+	spaceport "	At the center of the circle there's a terminal where people gather before they head off to their next destination. The ground level is a transport hub, where ground vehicles come to take or drop off people before going to the city. Close by, there's a large terraced garden with local flora, placed in a position where people can watch the ships take off."
```

Comment:
sanctuary will be used for missions where someone needs to escape to a place and need a lift. many people hunted will pay a fortune to escape there including possible criminals.

Reply:
Do you want to add those to pirate planets as well?

Reply:
yeah. would make sense, austral too